doc/osi-sensor-model-packaging_spec.adoc

```diff
@@ -0,0 +1,56 @@
+= OSI Sensor Model Packaging
+:revnumber: v.v.v 
+:revdate: dd.mm.yyyy
+:toc: left 
+:toclevels: 2
+:sectnums:
+
+:topicdir: topics
+:reusedir: reuse
+
+// header & footer for html
+// html themes
+//
+// source highlighting
+:source-highlighter: rouge
+:rouge-style: base16.solarized.light
+// 
+// :nofooter:
+:last-update-label!:
+//embed images into html
+:data-uri: 
+:mathematical-format: svg
+:imagesoutdir: ../images/generated_images
+
+include::./misc/idea_osmp.adoc[leveloffset=+1]
+
+== OSMP specification
+
+include::./spec/model_types.adoc[leveloffset=+2]
+
+include::./spec/basic_conventions.adoc[leveloffset=+2]
+
+include::./spec/binary_variables.adoc[leveloffset=+2]
+
+include::./spec/sensor_view_inputs.adoc[leveloffset=+2]
+
+include::./spec/sensor_view_input_configuration.adoc[leveloffset=+2]
+
+include::./spec/sensor_view_outputs.adoc[leveloffset=+2]
+
+include::./spec/sensor_data_inputs.adoc[leveloffset=+2]
+
+include::./spec/sensor_data_outputs.adoc[leveloffset=+2]
+
+include::./spec/ground_truth_init_parameters.adoc[leveloffset=+2]
+
+include::./spec/traffic_command_inputs.adoc[leveloffset=+2]
+
+include::./spec/traffic_update_outputs.adoc[leveloffset=+2]
+
+include::./spec/future_evolution.adoc[leveloffset=+2]
```

Comment:
This is not up to date to branch 68_migrate_spec_topics_to_asciidoc.
Currently it does not build due to "future_evolution.adoc" missing.